src/Kernel/CompiledCode.class.st

```diff
@@ -78,6 +78,13 @@ CompiledCode >> accessesField: varIndex [
 	^ (self readsField: varIndex) or: [ self writesField: varIndex ]
 ]
 
+{ #category : #scanning }
+CompiledCode >> accessesRef: literalVariable [
```

Comment:
I think it needs a better name #accessesLiteralVariable: . And same for readsRef: and writesRef: